### DISCUSSION TITLE
Should we add a version switcher for the docs?

### DISCUSSION THREAD

**Discussion by purva-thakre** [unitaryfund/mitiq#2383] [Ideas]
https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/version-dropdown.html

RTD allows us to switch between earlier versions of mitiq's documentation. The ability to switch between earlier versions is not as easily accessible unless we scroll down to the bottom of each page and look at the bottom left corner. 
![image](https://github.com/unitaryfund/mitiq/assets/66048318/c305665f-4ab0-4e2a-b331-d6220137ba15)

This option is also not available on certain pages like https://mitiq.readthedocs.io/en/stable/

With the PyData theme, there is a convenient option of using a [version switcher](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/version-dropdown.html) in the header/navigation bar. 

![image](https://github.com/unitaryfund/mitiq/assets/66048318/abcfea93-cf37-4f86-9909-58954aedb8a0)

**Comment by cosenal** [comment]
In my opinion the answer is _yes_, and I don't see much room for discussion 😄 I created https://github.com/unitaryfund/mitiq/issues/2386 and closing this.
